certdb/ocspstapling/ocspstapling.go

```diff
@@ -0,0 +1,117 @@
+package ocspstapling
+
+import (
+	"crypto"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/asn1"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"github.com/cloudflare/cfssl/certdb"
+	cferr "github.com/cloudflare/cfssl/errors"
+	"github.com/cloudflare/cfssl/helpers"
+	"github.com/google/certificate-transparency/go"
+	"golang.org/x/crypto/ocsp"
+)
+
+// OID of the SCT extension
+var sctExtOid = asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 11129, 2, 4, 2}
+
+// StapleSCTList inserts a list of Signed Certificate Timestamps into all OCSP
+// responses in a database wrapped by a given certdb.Accessor.
+//
+// NOTE: This function is patterned after the exported Sign method in
+// https://github.com/cloudflare/cfssl/blob/master/signer/local/local.go
+func StapleSCTList(acc certdb.Accessor, serial, aki string, scts []ct.SignedCertificateTimestamp, issuer *x509.Certificate, priv crypto.Signer) error {
+	// Grab all OCSP records that match serial and aki
+	ocspRecs, err := acc.GetOCSP(serial, aki)
+	if err != nil {
+		return err
+	}
+
+	if len(ocspRecs) == 0 {
+		// { OCSPRecord does not exist }
```

Comment:
We don't use them anywhere else in the code; it would make more sense to just do the assertion itself, or to write it as an actual comment (e.g. "At this point, we know the OCSP record doesn't exist.") Here, I don't think the comment is needed: we can see no records were returned, so the assumption is there's no record.